R/path.R

```diff
@@ -68,7 +60,7 @@ standardise_path <- function(path) {
     }
```

Comment:
I can't seem to leave a comment high enough, but how do you feel about `chr_to_file()`? It returns a freshly created file path, but never calls `enc2utf8()`. Should it? It creates a `tempfile()`, so I assume if you had a `User` name that included Unicode in it, then that file path would need to be converted to UTF-8

Reply:
I had been assuming this was OK as is, because I assume that `tempfile()` won't return a path with non-ascii characters. But I suppose you're right that it is technically possible that `tempdir()` could contain non-ascii characters.